ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,21 @@
+I'm submitting a ... (check one with "x")
+- [ ] bug report
+- [ ] feature request
+
+***For bug reports, please include:***
+
+Minimal SQL file that can be loaded into a clean database:
+``` sql
+
+```
+
+PostGraphQL version (CLI: `postgraphql --version`; Library: package version):
```

Comment:
Move this above the "For bug reports" line - I pretty much always want to know what version they're running 👍